ci/envs/39-pd12-conda-forge.yaml

```diff
@@ -2,6 +2,7 @@ name: test
 channels:
   - conda-forge
 dependencies:
+  - numpy <1.24 # compat with matplotlib 3.3.2
```

Comment:
While it is no longer needed, it might actually make sense to pin numpy here to an older version (eg of around the same age as pandas 1.2) to have some variety in numpy versions as well

Reply:
I've now pinned this to numpy 1.20 which came out 1 month after pandas 1.2 (on Jan 31 2021). This is dropped from NEP 29 support on Jan 31 2023 - in a month, but I suppose that's fine, if anything I suppose it's a prompt to update the pandas minimum version.

Reply:
Yes, something to discuss next year